Jenkinsfile

```diff
@@ -100,11 +136,16 @@ node('lisk-explorer-01'){
       sh '''#!/bin/bash
           # Stop Lisk-Node
           bash ~/lisk-test/lisk.sh stop
-          
+
           # Stop Lisk-Explorer
           pkill -f $(pwd)/app.js || true
+
+          # Cleanup e2e processes
+          pkill -f selenium -9 || true
```

Comment:
we should create a cleanup function instead for all of these cleanup things (my fault)